docs/diagrams/UiClassDiagram.puml

```diff
@@ -38,6 +38,8 @@ package Logic <<Rectangle>> {
 Class HiddenLogic #FFFFFF
 }
 
+Enum KeyboardShortcutsEnum <<enumeration>>
```

Comment:
I thought we agreed on removing this?

Reply:
Sorry, I forgot about this until the latest PR you made. The enum has been sitting inside the diagram for quite some time I was more concerned about the correctness of the diagram than the above issue.